test/quicapitest.c

```diff
@@ -379,7 +379,9 @@ static int test_ssl_trace(void)
     if (!TEST_ptr(cctx)
             || !TEST_ptr(bio)
             || !TEST_true(qtest_create_quic_objects(libctx, cctx, NULL, cert,
-                                                    privkey, 0, &qtserv,
+                                                    privkey,
```

Comment:
This doesn't make sense to me. We never call `qtest_add_time` anywhere here, so how does this work? It basically just makes the QUIC engine think the entire connection is taking place in the same instant of time, unless I'm missing something. Any timeout-based logic like loss detection or RTT estimation isn't going to work properly. This might make tests pass but it does so because it's not testing things properly. Seems broken unless I'm missing something.

Reply:
We are calling qtest_add_time(1) on line 316. I can probably move it out of the `if` to make it invoked unconditionally in the loop.

Reply:
Huh? You are calling `ossl_quic_conn_set_override_now_cb` in `qtest_create_quic_objects`. That overrides time retrieval for the entire QUIC stack. The CHANNEL and ACKM are not going to see accurate time.

Reply:
Yes, and how does it matter for ssl_trace_test() and test_corrupted_data()? This one was already using fake time on server.

Reply:
Ah, right, OK. This is just for the trace test. OK.